Kinvey.Tests/BaseTestClass.cs

```diff
@@ -110,20 +110,17 @@ public virtual void Tear()
         {
             try
             {
-                var client = Client.SharedClient;
+                var client = Client._sharedClient;
```

Comment:
What is the reasoning for not using the **SharedClient** property?

Reply:
`SharedClient` is just a compute property that throws an exception if the `_sharedClient` is `null`, so that's why we need to access the real property directly